src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -422,6 +424,18 @@ class PolicyHandler : public PolicyHandlerInterface,
    */
   uint32_t GetAppIdForSending() const OVERRIDE;
 
+  /**
+   * @brief Add application to PTU queue if don't have application
+   * with the same app_id
+   * @param new_app_id app id new application
+   */
+  void PushAppIdToQueue(const uint32_t new_app_id);
+
+  /**
+   * @brief Remove first from queue application
```

Comment:
@ydementieiev `Remove the first application from applications queue`

Reply:
@AByzhynar  update description [c04c115ce](https://github.com/smartdevicelink/sdl_core/commit/c04c115ced5d0ea1dfcc5bb53fc9c01d6d09a998)